src/java/org/apache/cassandra/service/accord/AccordConfigurationService.java

```diff
@@ -360,7 +360,7 @@ private static boolean shareShard(Topology current, Node.Id target, Node.Id self
         return false;
     }
 
-    public synchronized void onNodesRemoved(long epoch, Topology current, Set<Node.Id> removed)
+    public void onNodesRemoved(long epoch, Topology current, Set<Node.Id> removed)
```

Comment:
`syncPropagator` manages its own locks, so can ignore.

`nonCompletedEpochsBefore` and `receiveRemoteSyncCompletePreListenerNotify` also manage their own locks